<launch>
  <arg name="bias_file" default=""/>
  <node pkg="metavision_driver" type="sync_test" name="sync_test" clear_params="true"
	output="screen">
    <param name="use_ros_time" value="false"/>
    <remap from="~events_cam_0" to="/event_cam_0/events"/>
    <remap from="~events_cam_1" to="/event_cam_1/events"/>
  </node>
</launch>
